### PR TITLE
[1LP][RFR] fix network checks for pod appliance which currently has different db and http hosts

### PR DESCRIPTION
**Pull request by izapolsk**

### CODE REVIEW

File: cfme/test_framework/appliance_police.py [hunk 1]
```diff
@@ -28,36 +28,39 @@ def appliance_police():
...
-            'https': store.current_appliance.ui_port,
-            'postgres': ports.DB}
-        port_results = {pn: net_check(pp, force=True) for pn, pp in port_numbers.items()}
+        appliance = store.current_appliance
+        available_ports = {
+            'ssh': (appliance.hostname, ports.SSH),
+            'https': (appliance.hostname, appliance.ui_port),
+            'postgres': (appliance.db_host or appliance.hostname, ports.DB)}
+        port_results = {pn: net_check(addr=pp[0], port=pp[1], force=True)
+                        for pn, pp in available_ports.items()}
```
Comment (mfalesni) [inline cfme/test_framework/appliance_police.py]: **OPTIONAL**: `pn, (p_addr, _p_port)`

File: cfme/test_framework/appliance_police.py [hunk 2]
```diff
@@ -28,36 +28,39 @@ def appliance_police():
...
         return
     try:
-        port_numbers = {
-            'ssh': ports.SSH,
-            'https': store.current_appliance.ui_port,
-            'postgres': ports.DB}
-        port_results = {pn: net_check(pp, force=True) for pn, pp in port_numbers.items()}
+        appliance = store.current_appliance
+        available_ports = {
+            'ssh': (appliance.hostname, ports.SSH),
```
Comment (mfalesni) [inline cfme/test_framework/appliance_police.py]: **REQUIRED**: `appliance.ssh_port`

File: cfme/test_framework/appliance_police.py [hunk 3]
```diff
@@ -28,36 +28,39 @@ def appliance_police():
...
+            'https': (appliance.hostname, appliance.ui_port),
+            'postgres': (appliance.db_host or appliance.hostname, ports.DB)}
```
Comment (mfalesni) [inline cfme/test_framework/appliance_police.py]: **REQUIRED**: `appliance.db_port`

File: cfme/test_framework/appliance_police.py [hunk 4]
```diff
@@ -28,36 +28,39 @@ def appliance_police():
     if not store.slave_manager:
         return
     try:
-        port_numbers = {
-            'ssh': ports.SSH,
-            'https': store.current_appliance.ui_port,
-            'postgres': ports.DB}
-        port_results = {pn: net_check(pp, force=True) for pn, pp in port_numbers.items()}
+        appliance = store.current_appliance
```
Comment (RonnyPfannschmidt) [inline cfme/test_framework/appliance_police.py]: **required** use the appliance fixture

File: cfme/test_framework/appliance_police.py [hunk 5]
```diff
@@ -28,36 +28,39 @@ def appliance_police():
...
+        available_ports = {
```
Comment (RonnyPfannschmidt) [inline cfme/test_framework/appliance_police.py]: **note** this code looks a bit hard to read to me, but off hand there doesn't seem to be a better way to do it

**Comment by mshriver** [comment]
Waiting on PRT

**Pull request merged**